modules/ROOT/pages/starting-and-stopping-mule-esb.adoc

```diff
@@ -0,0 +1,180 @@
+| `install` | *(Windows only.)* Install Mule as a Windows service.
+| `remove` | *(Windows only.)* Remove Mule from your Windows services.
+|===
+
+== Running Mule
+
+You can test if Mule runs in your system without errors by running the following commands:
+
+* On Windows environments:
++
+[source,powershell]
+----
+$ $MULE_HOME\bin\mule.bat
+----
+
+* On Linux/Unix environments:
++
+[source,console]
+----
+$ $MULE_HOME/bin/mule
+----
+
+These commands run Mule in foreground mode, and the startup script displays information on the terminal’s standard output. You can’t issue further commands on the terminal as long as Mule is running.
+
+To stop Mule, press `CTRL-C` in the terminal in which the script is running.
+
+== Starting Mule as a Windows Service
+
+To run Mule as a Windows service, you need to install it first by running:
+
+[source,powershell]
+----
+$ $MULE_HOME\bin\mule.bat install
+----
+
+Once installed, you can run Mule as a service:
+
+[source,powershell]
+----
+$ $MULE_HOME\bin\mule.bat start
+----
+
+[NOTE]
+====
+When Windows restarts, the Mule service stops the same way as using the `mule stop` command.
+The only situation where this could be different is if Windows kills the process due to a timeout.
+====
+
+=== Adding Properties to wrapper.conf File After Installation
+
+After the installation of Mule as a Windows service, you can override the properties defined in the `wrapper.conf` file by declaring new properties in the `wrapper.additional.conf` file.
+
+The behavior changes depending on the Mule edition you are using:
+
+* Mule Community Edition
++
+. After adding a new property, reinstall or restart the Windows Service.
+. Add the property and ensure that its index is greater than the greatest index in the `wrapper-additional.conf` file. Otherwise, your property can be overridden.
++
+For example, if the `wrapper-additional.conf` has the property:
++
+`wrapper.java.additional.n=-Dproperty=value`
++
+When `n` is the greatest index, you should add your property in the `wrapper.conf` file incrementing the index:
++
+`wrapper.java.additional.n+1=-DyourProperty=valueOfYourProperty`
+
+* Mule Enterprise Edition
++
+. By default, you can add  20 extra properties after installing Mule as a Service.
+. Configure the number of extra properties you need, using the following command line argument
+when you install the service:
++
+`-additionalJavaProperties=numberOfExtraProperties`
+
+== Starting Mule as a Linux/Unix Daemon
+
+To run Mule as a Linux-Unix Daemon, you need to install it first by running:
+
+[source,console]
+----
+$ $MULE_HOME/bin/mule install
+----
+
+Once installed, you can run Mule as a daemon:
+
+[source,console]
+----
+$ $MULE_HOME/bin/mule start
+----
+
+[NOTE]
+====
+You can also start Mule without executing the installation step. This will start the Mule in the background for the current user session.
```

Comment:
"This will start the Mule" should be "This starts Mule" as we want to avoid future tense unless needed, and Mule is a proper noun, so we don't need the article "the".